CGPs/cgp-0060.md

```diff
@@ -256,6 +267,11 @@ The setup of the liquidity pool is based on the assumption that the cUSD/USDC pr
 
 The cUSD/USDCet pool will offer cUSD (and USDCet) at an incorrect price that is either higher (cUSD/USDCet < 1) or lower (cUSD/USDCet > 1) than the market price. The availability of expensive/cheap liquidity can have a price impact that is undesirable for cUSD as it is purely caused by a USDCet price deviation.
 
+ 
+## Market Interference
+Whenever arbitrageurs have matched the the prices on other pools with the StableSwap cUSD/USDCe price, most users will trade with the reserve StableSwap pool since it has lower fees and slippage especially for larger trades.
+In other words DEXs have to compete with reserve liquidity. This is a dynamic which we would like to avoid in general. In the current situation we believe that the benefit for the entire Celo ecosystem outweighs the risk of a competitive market intervention.
```

Comment:
```suggestion
In other words, cUSD/USDCet pools on other DEXs will have to compete with the reserve liquidity provided via this CGP. This is a dynamic which we would like to avoid in general. In the current situation we however believe that the benefit of governance-owned cUSD/USDCet liquidity for the entire Celo ecosystem outweighs the drawbacks.
```